pkg/cmd/pr/create/create_test.go

```diff
@@ -880,26 +832,21 @@ func Test_determineTrackingBranch_hasMatch(t *testing.T) {
 		},
 	}
 
-	cs.Stub("") // git config --get-regexp (ReadBranchConfig)
-	cs.Stub(`deadbeef HEAD
-deadb00f refs/remotes/origin/feature
-deadbeef refs/remotes/upstream/feature`) // git show-ref --verify (ShowRefs)
```

Comment:
The contents of this stub was unnecessarily changed in the rewrite. I will revert it to its original value